tests/e2e/README.md

```diff
@@ -10,4 +10,15 @@ If the chain state from `state.go` is modified, the `ChainStateWithProposalTypes
 
 When adding a new proposal type:
 * add a case for your proposal type to `json_utils.go/UnmarshalProposalWithType`
-* add a generator for your proposal type in `state_rapid_test.go` and add it to the `GetProposalGen` function 
\ No newline at end of file
+* add a generator for your proposal type in `state_rapid_test.go` and add it to the `GetProposalGen` function
+
+## Regenerating Traces
+
+The traces in `tracehandler_testdata` are generated by the test `trace_handlers_test.go/TestWriteExamples`.
+
+You can regenerate them by running `./regenerate_traces.sh/`.
```

Comment:
I think the `regenerate_traces.sh` file wasn't committed

Reply:
Thanks!